tests/xscope_device/src/test.xc

```diff
@@ -1,11 +1,12 @@
-// Copyright 2016-2021 XMOS LIMITED.
+// Copyright 2016-2024 XMOS LIMITED.
 // This Software is subject to the terms of the XMOS Public Licence: Version 1.
 #include <xs1.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include "control.h"
-#include "control_transport.h"
-#include "control_host_support.h"
+#include "control_shared.h"
+#include "control_transport_shared.h"
+#include "../../../lib_device_control/host/control_host_support.h"
```

Comment:
Relative path? Fix this in the buildfiles

Reply:
I will update here and in the rest of the tests.